packages/shared-component--hero/package.json

```diff
@@ -23,7 +23,8 @@
   },
   "peerDependencies": {
     "@coopdigital/foundations-vars": ">= 3.1.2 < 4",
-    "@coopdigital/shared-component--signpost": ">= 2.3.0 < 3"
+    "@coopdigital/shared-component--signpost": ">= 2.3.0 < 3",
+    "@coopdigital/shared-component--link": ">=1.4.2"
```

Comment:
Sorry, one little comment.

Every `peerDependencies` entry should also be in `devDependencies` which makes sure **npm link** works in other scenarios (local development, linked project/workspace).

We've missed signpost from a previous PR, but should add both signpost and link.

Reply:
My bad, will have a scan through and update them all